modules/swagger-codegen/src/main/resources/JavaPlayFramework/build.mustache

```diff
@@ -6,11 +6,5 @@ lazy val root = (project in file(".")).enablePlugins(PlayJava)
 
 scalaVersion := "2.11.7"
 
-libraryDependencies ++= Seq(
-javaJdbc,
-cache,
-javaWs{{#useSwaggerUI}},
-"org.webjars" % "swagger-ui" % "2.2.10-1"{{/useSwaggerUI}}{{#useBeanValidation}},
-"javax.validation" % "validation-api" % "1.1.0.Final"
-{{/useBeanValidation}}
-)
+libraryDependencies += {{#useSwaggerUI}}"org.webjars" % "swagger-ui" % "2.2.10-1"{{/useSwaggerUI}}
```

Comment:
This is not good. If you don't use swaggerUI or beanValidation, it will show `libraryDependencies += `
And it will not work. 
We should stick with the `Seq` notation which is good practice and give the ability to concatenate multiple dependencies in an array.

Finally, the other dependencies are there by default because they are part of the "default sample app" created by play framework which this generator is based on. I think that even if the generated code is not using them by default, most if not all users will use them. If they don't, they can always delete them.